lstchain/high_level/hdu_table.py

```diff
@@ -34,6 +34,16 @@
 
 wobble_offset = 0.4 * u.deg
 
+LST_EPOCH = Time('2018-10-01T00:00:00', scale='utc')
+
+
+def time_to_fits(time, epoch=LST_EPOCH):
```

Comment:
why "time_to_fits"?

Reply:
Because ot converts an astropy time to the fits representation